spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -99,6 +101,91 @@ private SpoonRunner(String title, File androidSdk, File applicationApk, File ins
     }
   }
 
+  /** Returns {@code false} if a test failed on any device. */
+  static boolean parseOverallSuccess(SpoonSummary summary) {
```

Comment:
Sure thing,  i just applied the square code style,  can you point me to some intelij code style i can run trough the commits?